apps/sms/views/shared/js/settings.js

```diff
@@ -35,14 +36,16 @@ var Settings = {
     var keyHandlerSet = {
       'dom.mms.operatorSizeLimitation': this.initMmsSizeLimitation.bind(this),
       'operatorResource.sms.maxConcat':
-        this.initSmsMaxConcatenatedMsg.bind(this)
+        this.initSmsMaxConcatenatedMsg.bind(this),
+      'messages.mms.sendReadReport.enabled':
```

Comment:
you need to use `[this.SEND_READ_REPORT_KEY]`, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Computed_property_names